src/Spatial.Tests/Euclidean/Vector2DTests.cs

```diff
@@ -196,6 +196,16 @@ public void ToDenseVector()
             Assert.AreEqual(2, actual[1]);
         }
 
+        [TestCase("1, 0")]
+        [TestCase("1, 1")]
+        [TestCase("1, -1")]
+        public void Orthogonal(string vs)
+        {
+            var v = Vector2D.Parse(vs);
+            var orthogonal = v.Orthogonal;
+            Assert.IsTrue(orthogonal.DotProduct(v) < 1e-6);
```

Comment:
I prefer passing in a string with the expected vector and do `Assert.AreEqual(Vector2D.Parse(expected), orthogonal)` here.